src/mqtt/mqtt_test.c

```diff
@@ -46,6 +45,19 @@
 
 /*-----------------------------------------------------------*/
 
+/* Wrapper to backward compatible for old version coreMQTT. */
+#if !defined MQTT_TEST_COREMQTT_LIBRARY_VERSION
+    #error "Can't get current MQTT library version"
+
+#else
+    #if MQTT_TEST_COREMQTT_LIBRARY_VERSION == "v1.2.0"
```

Comment:
I will suggest to use the version macro compare in code directly. Reader will know the difference better.